src/views/Register.vue

```diff
@@ -0,0 +1,214 @@
+        >
+          {{ $t('login.register') }}
+        </v-btn>
+      </v-col>
+    </v-row>
+  </v-form>
+</template>
+
+<script>
+import {minLength, required} from 'vuelidate/lib/validators';
+import User from '../store/user';
+
+export default {
+  name: 'Register',
+  data() {
+    return {
+      name: '',
+      user: new User('', ''),
+      submitted: false,
+      successful: false,
+      message: '',
+    };
+  },
+  computed: {
+
+    loggedIn() {
+      return this.$store.state.auth.status.loggedIn;
+    },
+
+    //  created() {
+    //   if (this.loggedIn) {
+    //     this.$router.push('/profile');
+    //   }
+    // },
+
+    nameErrors() {
+      const errors = [];
+      if (!this.$v.name.$dirty) {
+        return errors;
+      }
+      if (!this.$v.name.minLength) {
+        errors.push(this.$t('login.error-name-min-length'));
+      }
+      if (!this.$v.name.required) {
+        errors.push(this.$t('login.error-name-required'));
+      }
+      return errors;
+    },
+
+    emailErrors() {
+      const errors = [];
+      if (!this.$v.name.$dirty) {
+        return errors;
+      }
+      if (!this.$v.name.minLength) {
+        errors.push(this.$t('login.error-email'));
+      }
+      if (!this.$v.name.required) {
+        errors.push(this.$t('login.error-email'));
+      }
+      return errors;
+    },
+
+    passwordErrors() {
+      const errors = [];
+      if (!this.$v.name.$dirty) {
+        return errors;
+      }
+      if (!this.$v.name.minLength) {
+        errors.push(this.$t('login.error-password-min-length'));
+      }
+      if (!this.$v.name.required) {
+        errors.push(this.$t('login.error-password-required'));
+      }
+      return errors;
+    },
+  },
+  methods: {
+
+    handleLogin() {
+
+      this.loading = true;
+      this.$validator.validateAll().then(isValid => {
+        if (!isValid) {
+          this.loading = false;
+          return;
+        }
+
+        if (this.user.username && this.user.password) {
+
+          this.$store.dispatch('auth/login', this.user).then(
+              () => {
+
+                const user = {
+                  name: this.user.username,
+                };
+                this.$store.dispatch('user/setUser', user);
+                this.$router.push({name: 'Home'});
+              },
+              error => {
+                this.loading = false;
+                this.message =
+                    (error.response && error.response.data) ||
+                    error.message ||
+                    error.toString();
+                alert(this.$t('login.error-login'));
+              }
+          );
+        }
+      });
+    },
+
+    handleRegister() {
+      this.message = '';
+      this.submitted = true;
+      this.$validator.validate().then(isValid => {
+        console.log('chuj');
+        if (isValid) {
+          this.$store.dispatch('auth/register', this.user).then(
+              data => {
+                this.message = data.message;
```

Comment:
show a notification using notification module of vuex store, see usage at chat module